pv_site_api/_db_helpers.py

```diff
@@ -7,6 +7,7 @@
 """
 
 import datetime as dt
+from datetime import timezone
```

Comment:
CI says something wrong with this, 

Perhaps it should be 
- remove line 10
- use `dt.timezone.utc`

Reply:
> CI says something wrong with this,
> 
> Perhaps it should be
> 
> * remove line 10
> * use `dt.timezone.utc`

Oh yes, makes sense. Made the changes

Reply:
> > CI says something wrong with this,
> > Perhaps it should be
> > 
> > * remove line 10
> > * use `dt.timezone.utc`
> 
> Oh yes, makes sense. Made the changes

@peterdudfield can we run the tests again here?

Reply:
i just close the PR and open it again, I think you can do the same. 
I think we need to adjust CI, so it always runs